src/lib/functions.ts

```diff
@@ -109,13 +108,17 @@ export async function trigger({
 
     res?.send?.forEach((msg) => {
       const localServerProtocols = ['ws', 'wss', 'http', 'https']
-      const serverProtocol = parsedAsyncAPI.server(msg.server).protocol().toLowerCase()
-      const isBroadcast = localServerProtocols.includes(serverProtocol) && !isRemoteServer(parsedAsyncAPI, msg.server)
+      let isBroadcast: boolean = true
+      if (msg.server) {
+        const serverProtocol = parsedAsyncAPI.server(msg.server).protocol().toLowerCase()
+        isBroadcast = localServerProtocols.includes(serverProtocol) && !isRemoteServer(parsedAsyncAPI, msg.server)
+      }
       app.send(new GleeMessage({
         payload: msg.payload,
         headers: msg.headers,
         channel: msg.channel || message.channel,
         serverName: msg.server,
+        connection: message.connection,
```

Comment:
If no server name is passed then the message is sent to the connection of the incoming message.

Reply:
But this is always assigning the connection to the `message.connection`. Shouldn't we assign it ONLY when `msg.server` is not defined?